Create ui-c8y-1021-22-122-set-sticky-bottom-background-when-inside-dropdown-menu-10160.md (#3857)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-22-122-set-sticky-bottom-background-when-inside-dropdown-menu-10160.md

---
date: ""

> title: Set background color for sticky bottom when inside dropdown menu

Reviewer: ```suggestion
title: Fixed background color for sticky bottom element inside a dropdown menu
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-64956
version: 1021.22.122
---

> Set sticky-bottom background when inside dropdown menu (#10160)

Reviewer 2: ```suggestion
In certain scenarios, the sticky bottom element inside a dropdown menu had a transparent background color. This change sets the background color of the sticky bottom element to match the dropdown menu background color .
```

> In certain scenarios, the sticky bottom element inside a dropdown menu had a transparent background color. This change sets the background color of the sticky bottom element to match the dropdown menu background color .

Reviewer: ```suggestion
In certain scenarios, the sticky bottom element inside a dropdown menu had a transparent background color. This change sets the background color of the sticky bottom element to match the dropdown menu background color.
```
